### PR TITLE
Add support for multichild widgets to fn3

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

File: sky/packages/sky/lib/src/fn3/framework.dart [hunk 1]
```diff
@@ -248,6 +250,18 @@ abstract class Element<T extends Widget> implements BuildContext {
...
 
+  RenderObject get _descendantRenderObject {
+    RenderObject result;
+    void visit(Element element) {
+      if (element is RenderObjectElement)
+        result = element.renderObject;
+      else
+        visitChildren(visit);
+    }
+    visitChildren(visit);
```
Comment (Hixie) [inline sky/packages/sky/lib/src/fn3/framework.dart]: probably worth having asserts here (or somewhere) that visitChildren() for anything that isn't a RenderObjectElement only visits one child.

**Comment by Hixie** [comment]
LGTM but this really needs tests.

File: sky/packages/sky/lib/src/fn3/framework.dart (hunk 1 above)
Comment (abarth) [inline sky/packages/sky/lib/src/fn3/framework.dart]: Done.

**Comment by abarth** [comment]
Yeah, the next stop is tests for this and parent data nodes.

**Pull request merged**